Documentation/packaging.md

```diff
@@ -48,12 +48,17 @@ When the ownership and permissions of `/var/lib/rkt` are set up correctly, membe
 
 ### systemd units
 
-A few [example systemd unit files for rkt helper services][rkt-units] are included in the rkt sources. These units demonstrate a systemd-managed, socket-activated rkt [metadata-service][rkt-metadata-svc], along with a convenient periodic [garbage collection][rkt-gc] service invoked at 12-hour intervals to purge dead pods.
+A few [example systemd unit files for rkt helper services][rkt-units] are
```

Comment:
Please don't wrap lines in markdown source.

See https://github.com/coreos/docs/blob/master/STYLE.md#line-wrapping-considered-harmful